tests/ui/specs/VirtualList/VirtualGridList/ElementSize/VirtualGridList-GT-28446-specs.js

```diff
@@ -0,0 +1,62 @@
+const Page = require('../VirtualGridListPage');
+
+describe('Change Sapcing', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should change spotlight size when spaceing change [GT-28446]', function () {
+		// Check default item size.
+		const defaultItemWidth = Page.getItemSize().width;
+		const defaultItemHeight = Page.getItemSize().height;
+		const defaultSpacingWidth = Page.itemSpacing(0, 1).width;
+		const defaultSpacingHeight = Page.itemSpacing(0, 5).height;
+		Page.item(0).moveTo();
+		Page.spotlightSelect();
+		const defaultSpotlightWidth = Page.spotlightSize().width;
+		const defaultSpotlightHeight = Page.spotlightSize().height;
+		// Step 3: Knobs > VirtualGridList > spacing > 200
+		Page.inputSpacing.moveTo();
+		Page.spotlightSelect();
+		Page.backSpace();
+		Page.numPad(0);
+		Page.numPad(0);
+		Page.spotlightRight();
+		// Check item size when spacing increase.
+		const newItemWidth = Page.getItemSize().width;
+		const newItemHeight = Page.getItemSize().height;
+		const newSpacingWidth = Page.itemSpacing(0, 1).width;
+		const newSpacingHeight = Page.itemSpacing(0, 4).height;
+		Page.item(0).moveTo();
+		Page.spotlightSelect();
+		const newSpotlightWidth = Page.spotlightSize().width;
+		const newSpotlightHeight = Page.spotlightSize().height;
+		// Step 3-1 Verify: The gap between items grow bigger.
+		expect(newSpacingWidth).to.be.above(defaultSpacingWidth);
+		expect(newSpacingHeight).to.be.above(defaultSpacingHeight);
```

Comment:
How about verifying the exact value?
```suggestion
		// Step 3-1 Verify: The spacing should be 100 for FHD.
		expect(newSpacingWidth).to.be.equal(100);
		expect(newSpacingHeight).to.be.equal(100);
```

Reply:
Suggested change.